basyx.aasrepository/basyx.aasrepository-backend/pom.xml

```diff
@@ -19,6 +19,10 @@
 			<groupId>org.eclipse.digitaltwin.basyx</groupId>
 			<artifactId>basyx.aasrepository-core</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.eclipse.digitaltwin.basyx</groupId>
+			<artifactId>basyx.mongodbcore</artifactId>
```

Comment:
Why basyx.mongodbcore in aasrepository-backend?

Reply:
It was added by mistake, removed it.